src/test/java/com/google/firebase/messaging/FirebaseMessagingClientImplTest.java

```diff
@@ -200,14 +200,14 @@ public void testSendErrorWithZeroContentResponse() {
   @Test
   public void testSendErrorWithMalformedResponse() {
     for (int code : HTTP_ERRORS) {
-      response.setStatusCode(code).setContent("not json");
+      response.setStatusCode(code).setContent("[not json]");
```

Comment:
I wonder if a text that looks more like HTML can be more representative here, e.g., <html>"not json"</html> ?

Reply:
From my tests and inspection of the `JsonParser` source, we need more than just HTML tags to trigger this IllegalArgumentException. Square brackets seems to be the easiest and most reliable way. Although it makes me wonder what kind of response was served by FCM to cause this error in the first place.